proto/stream_service.proto

```diff
@@ -92,6 +92,7 @@ message BarrierCompleteResponse {
   }
   repeated GroupedSstableInfo synced_sstables = 4;
   uint32 worker_id = 5;
+  bool checkpoint = 6;
```

Comment:
Can you also document the fact this flag can be false even if it is corresponding to a checkpoint barrier and why?